src/worker_job.cc

```diff
@@ -160,7 +160,18 @@ namespace detail {
 				const auto [bid, mode] = access_map.get_nth_access(i);
 				const auto sr = grid_box_to_subrange(access_map.get_requirements_for_nth_access(i, tsk->get_dimensions(), data.sr, tsk->get_global_size()));
 				const auto info = m_buffer_mngr.access_host_buffer(bid, mode, sr);
-				access_infos.push_back(closure_hydrator::accessor_info{info.ptr, info.backing_buffer_range, info.backing_buffer_offset, sr});
+
+#if CELERITY_ACCESSOR_BOUNDARY_CHECK
+					std::vector<id<3>> oob_indices {2};
```

Comment:
Maybe add a comment here that `oob_indices[0]` carries minima and `oob_indices[1]` carries maxima.